skore/src/skore/sklearn/_cross_validation/metrics_accessor.py

```diff
@@ -100,8 +101,9 @@ def report_metrics(
             Whether to flatten the `MultiIndex` columns. Flat index will always be lower
             case, do not include spaces and remove the hash symbol to ease indexing.
 
-        aggregate : {"mean", "std"} or list of such str, default=None
+        aggregate : None or list of str, default=("mean", "std")
```

Comment:
It would be my prefer writing.

```suggestion
        aggregate : {"mean", "std}, list of such str or None, default=("mean", "std")
```

Reply:
updated all docstring